File: app/src/main/java/hegde/mahesh/avre/ui/startup_scripts/StartupScriptsViewModel.java
package hegde.mahesh.avre.ui.startup_scripts;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class StartupScriptsViewModel extends ViewModel {

    public static final String TO_BE_IMPLEMENTED = "To be implemented";
    private final MutableLiveData<String> mText;

    public StartupScriptsViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue(TO_BE_IMPLEMENTED);
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: app/src/main/java/hegde/mahesh/avre/ui/startup_scripts/StartupScriptsViewModel.kt
package hegde.mahesh.avre.ui.startup_scripts

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class StartupScriptsViewModel : ViewModel() {
    private val mText: MutableLiveData<String> = MutableLiveData()

    init {
        mText.value = TO_BE_IMPLEMENTED
    }

    val text: LiveData<String>
        get() = mText

    companion object {
        const val TO_BE_IMPLEMENTED = "To be implemented"
    }
}